@Yajo 

```
2020-03-06 13:20:52,245 6833 INFO openerp_test odoo.tests.common: Terminating phantomjs
2020-03-06 13:20:52,250 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: FAIL
2020-03-06 13:20:52,250 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ======================================================================
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: FAIL: test_contact_unsubscription (odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui.UICase)
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` Test a mass mailing contact that wants to unsubscribe.
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: Traceback (most recent call last):
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 362, in phantom_poll
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.fail(pformat(json.loads(line[prefix:])))
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/__init__.py", line 319, in loads
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     return _default_decoder.decode(s)
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/decoder.py", line 339, in decode
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/decoder.py", line 357, in raw_decode
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     raise JSONDecodeError("Expecting value", s, err.value) from None
2020-03-06 13:20:52,251 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` During handling of the above exception, another exception occurred:
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` Traceback (most recent call last):
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/build/OCA/social/mass_mailing_custom_unsubscribe/tests/test_ui.py", line 90, in test_contact_unsubscription
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     ready=self._tour_ready % tour)
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 459, in phantom_js
2020-03-06 13:20:52,252 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.phantom_run(cmd, timeout)
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 386, in phantom_run
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     result = self.phantom_poll(phantom, timeout)
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 364, in phantom_poll
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.fail(lline)
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` AssertionError: error timeout after 10000 ms
2020-03-06 13:20:52,253 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ======================================================================
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: FAIL: test_partner_unsubscription (odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui.UICase)
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` Test a partner that wants to unsubscribe.
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: Traceback (most recent call last):
2020-03-06 13:20:52,253 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 362, in phantom_poll
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.fail(pformat(json.loads(line[prefix:])))
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/__init__.py", line 319, in loads
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     return _default_decoder.decode(s)
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/decoder.py", line 339, in decode
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/opt/python/3.5.6/lib/python3.5/json/decoder.py", line 357, in raw_decode
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     raise JSONDecodeError("Expecting value", s, err.value) from None
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` During handling of the above exception, another exception occurred:
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` Traceback (most recent call last):
2020-03-06 13:20:52,254 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/build/OCA/social/mass_mailing_custom_unsubscribe/tests/test_ui.py", line 134, in test_partner_unsubscription
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     ready=self._tour_ready % tour)
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 459, in phantom_js
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.phantom_run(cmd, timeout)
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 386, in phantom_run
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     result = self.phantom_poll(phantom, timeout)
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `   File "/home/travis/OCB-11.0/odoo/tests/common.py", line 364, in phantom_poll
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: `     self.fail(lline)
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: ` AssertionError: error tour mass_mailing_custom_unsubscribe_tour_partner failed at step .radio:contains('other reason') :radio:not(:checked)
2020-03-06 13:20:52,255 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: Ran 2 tests in 25.187s
2020-03-06 13:20:52,255 6833 ERROR openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui: FAILED
2020-03-06 13:20:52,255 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui:  (failures=2)
2020-03-06 13:20:52,256 6833 INFO openerp_test odoo.modules.module: odoo.addons.mass_mailing_custom_unsubscribe.tests.test_ui tested in 25.19s, 1716 queries
2020-03-06 13:20:52,256 6833 ERROR openerp_test odoo.modules.module: Module mass_mailing_custom_unsubscribe: 2 failures, 0 errors
2020-03-06 13:20:52,256 6833 INFO openerp_test odoo.modules.module: odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription running tests.
2020-03-06 13:20:52,257 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: test_details_required (odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription.UnsubscriptionCase)
2020-03-06 13:20:52,258 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: ` Cannot create unsubscription without details when required.
2020-03-06 13:20:52,286 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: test_reason_required (odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription.UnsubscriptionCase)
2020-03-06 13:20:52,286 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: ` Cannot create unsubscription without reason when required.
2020-03-06 13:20:52,308 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: Ran 2 tests in 0.051s
2020-03-06 13:20:52,308 6833 INFO openerp_test odoo.addons.mass_mailing_custom_unsubscribe.tests.test_unsubscription: OK
2020-03-06 13:20:52,872 6833 INFO openerp_test odoo.modules.registry: module mass_mailing_newsletter_welcome_mail: creating or updating database tables
2020-03-06 13:20:52,909 6833 INFO openerp_test odoo.modules.loading: loading mass_mailing_newsletter_welcome_mail/data/mail_template.xml
2020-03-06 13:20:53,007 6833 INFO openerp_test odoo.modules.loading: loading mass_mailing_newsletter_welcome_mail/views/mail_mass_mailing_list.xml
2020-03-06 13:20:53,090 6833 INFO openerp_test odoo.modules.loading: 56 modules loaded in 75.68s, 28256 queries
2020-03-06 13:20:53,427 6833 ERROR openerp_test odoo.modules.loading: At least one test failed when loading the modules.
```

--- a/mail_debrand/models/mail_template.py
+++ b/mail_debrand/models/mail_template.py
@@ -21,7 +21,11 @@ class MailTemplate(models.Model):
 
     @api.model
     def _debrand_body(self, body):
-        using_word = _('using')
+        """Depending of the context the term could come translated or in
+           the original term. We should consider both. Also notice that if
+           the translated term in this module differs from the one in the
+           template it won't be catched."""
+        using_word = "(using|{})".format(_('using'))
         odoo_word = _('Odoo')
         return re.sub(
             using_word + "(.*)[\r\n]*(.*)>" + odoo_word + r"</a>", "", body,

